django/db/migrations/serializer.py

```diff
@@ -244,8 +244,9 @@ def _format(self):
 
 class SetSerializer(BaseSequenceSerializer):
     def _format(self):
-        # Don't use the literal "{%s}" as it doesn't support empty set
-        return "set([%s])"
+        # When value is empty, the empty set should be serialized as "set()",
+        # not "{}" because {} is an empty dict.
+        return '{%s}' if self.value else 'set(%s)'
```

Comment:
If we were to change this --- I don't have strong opinions on that --- I don't want to see this as a precedent for making migration files to adhere to flake8.

Given this particular line, it makes the code _slightly_ harder to read and needs a second line of comment. _I_ don't see the benefit here.

My 2 ¢